### DISCUSSION TITLE
Using a third party code generator will help x86_64 support possible?

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#794] [General]
Except LLVM, otherwise occ will now be just another frontend like clang. There is mir and various alternatives listed at the end of the page:

https://github.com/vnmakarov/mir

Well, you see. Even a toy compiler supports x86_64 nowadays: https://github.com/tyfkda/xcc

**Comment by GitMensch** [comment]
But that misses `goto` statement (and in fact is 64 bit GNU Linux or newer mac only)!

**Reply by ghost** [reply]
People told me `goto` is evil.

**Reply by GitMensch** [reply]
then people don't write code generators :-)

**Reply by ghost** [reply]
What about this? https://github.com/TinyCC/tinycc

**Comment by LADSoft** [comment]
code generation for x64 is actually one of my higher priorities now...   but it is stacked behind two other high-priority projects so it will be a while sigh...
